app/assets/js/rtm-upload-terms.js

```diff
@@ -1,139 +1,169 @@
+
+    /**
+     * When Select Attribute for media [ rtmedia-custom-attributes: Add-Ons ] Issue:8,
+     * This Hook returns false that's why the attributes were not saved,
+     * Added: 'return true;'
+     * By: Yahil
+     */
+    rtMediaHook.register( 'rtmedia_js_after_file_upload', function () {
+
+        var terms_conditions_checkbox = jQuery( '#rtmedia-upload-container #rtmedia_upload_terms_conditions' );
+        if ( 1 === terms_conditions_checkbox.length ) {
+            terms_conditions_checkbox.removeAttr( 'checked' );
+            jQuery( '.rt_alert_msg' ).remove();
+        }
+
+        return true;
+    });
+
+    /**
+     * Uncheck the terms checkbox after the activity is posted successfully.
+     */
+    rtMediaHook.register( 'rtmedia_js_after_activity_added', function () {
+        var rtmedia_terms_conditions = $( '#rtmedia_upload_terms_conditions' );
+        if ( 1 === rtmedia_terms_conditions.length && rtmedia_terms_conditions.is( ':checked' ) ) {
+            rtmedia_terms_conditions.prop( 'checked', false );
+        }
+    } );
 }
 
-jQuery(document).ready(function () {
-	var terms_conditions_checkbox = jQuery( '#rtmedia_upload_terms_conditions' );
-
-	if ( terms_conditions_checkbox.length ) {
-		terms_conditions_checkbox.on( 'click', function () {
-
-			// If start upload button exist, then focus to that button.
-			var upload_start_btn = jQuery('.start-media-upload');
-			if ( upload_start_btn.length ) {
-				upload_start_btn.focus();
-			}
-
-			// Show error message if terms-condition is not checked.
-			if ( terms_conditions_checkbox.is( ':checked' ) ) {
-				var alter_msg_span = terms_conditions_checkbox.siblings( 'span.rt_alert_msg' );
-				if ( 0 < alter_msg_span.length ) {
-					alter_msg_span.remove();
-				} else {
-					terms_conditions_checkbox.parent().siblings( 'span.rt_alert_msg' ).remove();
-				}
-
-			} else {
-				rtp_display_terms_warning( terms_conditions_checkbox.parent( '.rtmedia-upload-terms' ), rtmedia_upload_terms_data.message );
-			}
-
-			if ( typeof rtmedia_direct_upload_enabled !== 'undefined' && rtmedia_direct_upload_enabled == '1' ) {
-				var whats_new_submit = jQuery( '#aw-whats-new-submit' );
-				if ( whats_new_submit.length > 0 ) {
-					if ( jQuery( '#whats-new' ).val().trim() !== '' || jQuery( '#rtmedia_uploader_filelist' ).children( 'li' ).length > 0) {
-						whats_new_submit.trigger( 'click' );
-					}
-				} else {
-					jQuery( '.start-media-upload' ).trigger( 'click' );
-				}
-			}
-		});
-	}
-
-});
+jQuery( document ).ready( function () {
+
+    var terms_conditions_checkbox = jQuery( '#rtmedia_upload_terms_conditions' );
+    if ( 1 === terms_conditions_checkbox.length ) {
+        terms_conditions_checkbox.on( 'click', function () {
+
+            // If start upload button exist, then focus to that button.
+            var upload_start_btn = jQuery( '.start-media-upload' );
+            if ( 1 === upload_start_btn.length ) {
+                upload_start_btn.focus();
```

Comment:
Just write `jQuery( '.start-media-upload' ).focus()`.

Reply:
`upload_start_btn` variable is used in line 142.